launcher/launcher/main.go

```diff
@@ -72,6 +79,16 @@ func main() {
 		return
 	}
 
+	args := fmt.Sprintf("-output=%s", experimentDataFile)
```

Comment:
Does output assume a prefix of `launcherfile.HostTmpPath`? Or do you need to `path.Join` with `experimentDataFile` here? (if so, you can pull out the path.Join below into a variable)

Reply:
oopsie, it does need the combo. I appear to have rushed this commit :)